lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -330,6 +331,13 @@ dummy 1.2.3
             depName: 'php',
             extractVersion: '^php-(?<version>\\S+)',
           },
+          {
+            currentValue: '7.26.2',
+            datasource: 'npm',
+            packageName: 'pnpm',
+            depName: 'pnpm',
+            versioning: 'semver',
```

Comment:
@viceice apparently this is mandatory in the types, so a change/optimization for another time